zebra/zebra_vty.c

```diff
@@ -4298,6 +4299,26 @@ DEFUN_HIDDEN(no_zebra_kernel_netlink_batch_tx_buf,
 
 #endif /* HAVE_NETLINK */
 
+#ifdef HAVE_SCRIPTING
+
+DEFUN(set_scriptname, set_scriptname_cmd, "script HOOKCALL SCRIPT",
+      "Specify a script file for a hook call\n"
+      "Hook call function name\n"
+      "Script name (same as filename in /etc/frr/scripts/, without .lua)\n")
+{
```

Comment:
For the command, I think @qlyoung  wanted to leave it up to individual places to add the function/script info rather than an all-encompassing script command per daemon, right?

so like:

```
zebra on-dplane-install script myfunnyscript.lua
```

Reply:
Correct, we shouldn't need this command defined; the idea is to accept script names naturally where we would other configuration related to the function the script would serve, not to have a global mapping of script -> hook point.

Reply:
updated hook call specific command `zebra on-rib-process script SCRIPT`